src/irr/video/COpenGLFunctionTable.h

```diff
@@ -562,7 +582,7 @@ namespace irr {
 												GL_TEXTURE_1D_ARRAY,GL_TEXTURE_2D_ARRAY,GL_TEXTURE_BUFFER, // GL 3.x
 												GL_TEXTURE_CUBE_MAP_ARRAY,GL_TEXTURE_2D_MULTISAMPLE,GL_TEXTURE_2D_MULTISAMPLE_ARRAY }; // GL 4.x
 
-			if (COpenGLExtensionHandler::Version >= 440 || COpenGLExtensionHandler::FeatureAvailable[COpenGLExtensionHandler::EOpenGLFeatures::IRR_ARB_multi_bind])
+			if (COpenGLDriver::Version >= 440 || COpenGLDriver::FeatureAvailable[COpenGLDriver::EOpenGLFeatures::IRR_ARB_multi_bind])
```

Comment:
if you create that `COpenGLFeatureMap` class and move all the Version + Feature variables there, then you will no longer have the circular dependency, because here you'd be using `COpenGLFeatureMap::` wherever you're currently trying to use `COpenGLDriver::` and you wouldn't need to know anything about `COpenGLDriver`